manifests/agent/windows.pp

```diff
@@ -0,0 +1,40 @@
+# Installs the Google Cloud Logging Agent to a Windows machine.
+#
+# For more details please visit:
+#   https://cloud.google.com/logging/docs/agent/installation
+class glogging::agent::windows(
+  $version = 46,
+) {
+
+  $repo = 'https://repo.stackdriver.com/windows'
+  $installer = "StackdriverMonitoring-GCM-${version}.exe"
+  $download_url = "${repo}/${installer}"
+
+  $install_dir = 'C:\Program Files (x86)\Stackdriver\MonitoringAgent'
```

Comment:
Can we assume that they're on a 64bit machine?

Reply:
Yes. Windows servers are always 64-bits.